threestudio/models/guidance/deep_floyd_guidance.py

```diff
@@ -343,17 +343,17 @@ def guidance_eval(
         # use only 50 timesteps, and find nearest of those to t
         self.scheduler.set_timesteps(50)
         self.scheduler.timesteps_gpu = self.scheduler.timesteps.to(self.device)
-        bs = latents_noisy.shape[0]  # batch size
-        large_enough_idxs = self.scheduler.timesteps_gpu.expand(
-            [bs, -1]
-        ) > t_orig.unsqueeze(
+        bs = min(4, latents_noisy.shape[0])  # batch size
+        large_enough_idxs = self.scheduler.timesteps_gpu.expand([bs, -1]) > t_orig[
+            :bs
+        ].unsqueeze(
             -1
         )  # sized [bs,50] > [bs,1]
         idxs = torch.min(large_enough_idxs, dim=1)[1]
         t = self.scheduler.timesteps_gpu[idxs]
 
         fracs = list((t / self.scheduler.config.num_train_timesteps).cpu().numpy())
-        imgs_noisy = latents_noisy.permute(0, 2, 3, 1)
+        imgs_noisy = (latents_noisy[:bs] / 2 + 0.5).permute(0, 2, 3, 1)
```

Comment:
So the noisy_latents weren't scaled and biases properly for DeepFloyd, previously?

Reply:
Yes, none of the latents were. This is only during guidance_eval